chain-core/src/init/coin.rs

```diff
@@ -241,6 +241,39 @@ mod test {
     use quickcheck::quickcheck;
 
     quickcheck! {
+        // test whether oveflow error occur
```

Comment:
Quickcheck is meant for property tests where the argument to the test functions are fed with random inputs, so it's a bit confusing / redundant to have unit tests here.

it'll be better to move these up (from the quickcheck macro) as regular units in Rust:
```
    #[test]
    fn coin_overflow_add() {
    ...
    assert!(sum.is_err())
    }
```

and perhaps give the more meaningful names than `coin_add`, `coin_add2`...

For example: `add_2_coin_max_should_produce_error`

Reply:
done